sql/wire/http.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Vivek Menezes (vivek@cockroachlabs.com)
+
+package sqlwire
+
+const (
+	// SQLDBEndpoint is the URL path prefix which accepts incoming
+	// HTTP requests for the SQL API.
+	SQLDBEndpoint = "/sql/db/"
+	// StatusTooManyRequests indicates client should retry due to
+	// server having too many requests.
+	// Defined here because it's still not a part of the HTTP standard.
+	StatusTooManyRequests = 429
```

Comment:
This was in `client/http_sender.go`, but I'm not sure if the server was ever sending back this status code. Perhaps drop from this PR until we actually have code that uses it.